### PR TITLE
Add examples of individually setting bits in Physics introduction

### PR DESCRIPTION
**Pull request by D4Devil**
Added examples for independently setting bits on mask and layer

### CODE REVIEW

File: tutorials/physics/physics_introduction.rst [hunk 1]
```diff
@@ -159,7 +159,7 @@ Code example
 
 In function calls, layers are specified as a bitmask. Where a function enables
 all layers by default, the layer mask will be given as ``0xffffffff``. Your code
-can use binary, hexadecimal, or decimal notation for layer masks, depending
+can use binary, hexadecimal, or decimal notation for layer masks depending
```
Comment (AThousandShips) [inline tutorials/physics/physics_introduction.rst:162]: Please restore, unrelated and an appropriate pause

File: tutorials/physics/physics_introduction.rst [hunk 2]
```diff
@@ -180,6 +180,15 @@ would be as follows::
     # (2^(1-1)) + (2^(3-1)) + (2^(4-1)) = 1 + 4 + 8 = 13
     pow(2, 1-1) + pow(2, 3-1) + pow(2, 4-1)
 
+You can also set bits independently by calling 'set_collision_layer_value(layer_number, value)'
```
Comment (AThousandShips) [inline tutorials/physics/physics_introduction.rst:183]:
```suggestion
You can also set bits independently by calling ``set_collision_layer_value(layer_number, value)``
```

File: tutorials/physics/physics_introduction.rst [hunk 3]
```diff
@@ -180,6 +180,15 @@ would be as follows::
...
+or 'set_collision_mask_value(layer_number, value)' on any given :ref:`CollisionObject <class_CollisionObject2D>` as follows::
```
Comment (AThousandShips) [inline tutorials/physics/physics_introduction.rst:184]:
```suggestion
or ``set_collision_mask_value(layer_number, value)`` on any given :ref:`CollisionObject <class_CollisionObject2D>` as follows::
```

File: tutorials/physics/physics_introduction.rst [hunk 4]
```diff
@@ -180,6 +180,15 @@ would be as follows::
...
+
+    # Example: Setting mask value for enabling layers 1, 3 and 4
```
Comment (AThousandShips) [inline tutorials/physics/physics_introduction.rst:186]:
```suggestion
    # Example: Setting mask value for enabling layers 1, 3, and 4.
```

**Comment by D4Devil** [comment]
@AThousandShips Thanks for the corrections

File: tutorials/physics/physics_introduction.rst [hunk 5]
```diff
@@ -180,6 +180,15 @@ would be as follows::
     # (2^(1-1)) + (2^(3-1)) + (2^(4-1)) = 1 + 4 + 8 = 13
     pow(2, 1-1) + pow(2, 3-1) + pow(2, 4-1)
 
+You can also set bits independently by calling ``set_collision_layer_value(layer_number, value)``
+or ``set_collision_mask_value(layer_number, value)`` on any given :ref:`CollisionObject <class_CollisionObject2D>` as follows::
```
Comment (tetrapod00) [inline tutorials/physics/physics_introduction.rst:184]:
```suggestion
or ``set_collision_mask_value(layer_number, value)`` on any given :ref:`CollisionObject2D <class_CollisionObject2D>` as follows::
```
Use the whole name

File: tutorials/physics/physics_introduction.rst [hunk 6]
```diff
@@ -180,6 +180,15 @@ would be as follows::
...
+
+    # Example: Setting mask value to enable layers 1, 3, and 4.
+
+    var collider: CollisionObject2D = $CollisionObject2D  # Any given collider.
+    collider.set_mask_layer_value(1, true)
+    collider.set_mask_layer_value(3, true)
+    collider.set_mask_layer_value(4, true)
```
Comment (tetrapod00) [inline tutorials/physics/physics_introduction.rst:191]:
```suggestion
    collider.set_collision_mask_value(1, true)
    collider.set_collision_mask_value(3, true)
    collider.set_collision_mask_value(4, true)
```
The function name was wrong in the example, there is no `set_mask_layer_value()` function

**Review by tetrapod00 — CHANGES_REQUESTED** [review]
I tested the code (with the function name correction applied). It does work to set the bits of the collision mask one at a time.

**Review by tetrapod00 — APPROVED** [review]

**Pull request merged**